bench/tests/test_init.py

```diff
@@ -78,8 +78,8 @@ def test_new_site(self):
 	def new_site(self, site_name):
 		new_site_cmd = ["bench", "new-site", site_name, "--admin-password", "admin"]
 
-		# set in travis
-		if os.environ.get("TRAVIS"):
+		# set in CI
+		if os.environ.get('CI'):
```

Comment:
`if os.environ.get("TRAVIS") or os.environ.get('CI'):` would be better I guess, in case we do go back.